gateway/src/config/mod.rs

```diff
@@ -18,10 +18,10 @@ pub(crate) const MISSING_VALUE: &str = "MISSING VALUE";
 const DEFAULT_MIX_LISTENING_PORT: u16 = 1789;
 const DEFAULT_CLIENT_LISTENING_PORT: u16 = 9000;
 pub(crate) const DEFAULT_VALIDATOR_REST_ENDPOINTS: &[&str] = &[
-    "http://testnet-finney-validator.nymtech.net:1317",
-    "http://testnet-finney-validator2.nymtech.net:1317",
+    "http://testnet-milhon-validator1.nymtech.net:1317",
```

Comment:
I don't remember the details, but isn't it the case gateways are talking with validator API rather than `nymd`? Meaning that the port should probably be `8080`? - I don't know the answer for it, just asking. But if you run it and it worked, it's good enough for me.